<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>文件名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>路径</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>转换前格式</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>转换后格式</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>进度</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuAct">
    <property name="title">
     <string>动作</string>
    </property>
    <addaction name="action_UTF_8"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAct"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>清空列表</string>
   </property>
  </action>
  <action name="action_UTF_8">
   <property name="text">
    <string>转换为UTF-8</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
